crates/starknet-devnet-server/src/api/http/models.rs

```diff
@@ -41,6 +42,8 @@ pub struct MessageHash {
 pub struct TxHash {
     pub transaction_hash: TransactionHash,
 }
+// Implemented as type alias so JSON returned doesn't have extra key
+pub type Dump = Option<Vec<DumpEvent>>;
```

Comment:
I find this type misleading: a `Dump` should always be something. A dump response body, on the other hand, could be optional. It would be better for those functions whose return type is `SomeWrapper<Dump>` to rather rather return `SomeWrapper<Option<Vec<DumpEvent>>>`. Or perhaps just rename `Dump` to `DumpResponseBody`.